src/sentry/seer/anomaly_detection/detect_anomalies.py

```diff
@@ -0,0 +1,101 @@
+import logging
```

Comment:
this is the _start_ of pulling the `get_anomaly_data_from_seer` out into a separate helper that we can utilize in both the subscription_processor as well as the proxy api.

this is incomplete atm